docs/platform/webcontainers/browser-config.md

```diff
@@ -22,15 +22,30 @@ With Chrome's defaults, starting with version 118 or later, you should get a pre
 
 Read on if you run into issues or want an even smoother experience.
 
-Previews opened in a separate tab will require you to connect them to the editor. You usually get prompted to click on a button to do that. However, you might do that more frequently than expected with Chrome's [memory saver][CHROME_MEMORY_SAVER]. To avoid it, you can add the following exception in [chrome://settings/performance](chrome://settings/performance):
+### Previews in a separate tab {#chrome-preview-separate-tab}
+
+Previews that are opened in a separate tab are not connected by default to the editor and require an extra step. You usually get prompted to click a button to do that.
+
+You can have it done automatically by modifying your popups settings, [chrome://settings/content/popups](chrome://settings/content/popups), with the following exception:
+
+```
+https://[*.]webcontainer.io
+```
+
+<img class="mx-auto" alt="Chrome popups settings showing the *.webcontainer.io origins as exceptions" src="./assets/chrome-settings-popups.png" width="800" height="502.37" />
+
+Once done, whenever the preview need to reconnect to the editor, it will open a popup that gets immediately closed.
+
+Lastly, if you have Chrome's [memory saver][CHROME_MEMORY_SAVER] turned on, this step might be neccessary more often than required. To avoid it, you can add the following exception in [chrome://settings/performance](chrome://settings/performance):
```

Comment:
```suggestion
Lastly, if Chrome's [memory saver][CHROME_MEMORY_SAVER] is turned on, this step might be necessary more often than required. To avoid it, you can add the following exception in [chrome://settings/performance](chrome://settings/performance):
```

Also, what do you mean by "might be necessary more often than required"?